examples/lab_devices/sensirionSHT85.py

```diff
@@ -0,0 +1,33 @@
+#
+# ------------------------------------------------------------
+# Copyright (c) All rights reserved
+# SiLab, Institute of Physics, University of Bonn
+# ------------------------------------------------------------
+#
+
+from basil.dut import Dut
+
+import time
+
+dut = Dut('examples/lab_devices/sensirionSHT85.yaml')
+dut.init()
+dut['Thermohygrometer'].printInformation()
+print(dut['Thermohygrometer'].get_temperature_and_humidity())
```

Comment:
This can be added to the temperature control example file.. maybe we split this into 'temperature_sensors' and 'temperature_control' now, since it's getting quite big..

Reply:
actually, i will already implement this with #135 so you can add your code to the 'temperature_sensors' example file once it's merged..

Reply:
#135 is merged, so you can rebase and add your code to the 'termperature_sensors' example

Reply:
Done